tests/modules/accordion.spec.tsx

```diff
@@ -0,0 +1,122 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
```

Comment:
Could you put these files in the same folder format as the `src` directory. I.e. `tests/accordion/accordion.spec.tsx`

Same for the rest